ansible/roles/cfg-logging/tasks/main.yml

```diff
@@ -7,7 +7,5 @@
 
 - name: Place logging config file
   template: src={{src}} dest={{dest}} owner={{owner}} group={{group}} mode=0644
-  when: "'{{group_name}}' in group_names"
```

Comment:
If we are deploying logging files to the host machine, this protection is needed so that the file is placed ONLY on the appropriate machine.
If we are getting configs from a docker container, then this protection is no longer needed.
The same holds for the checks in `cfg-service/tasks/main.yml`